Jakefile

```diff
@@ -280,10 +289,43 @@ for (var i in libraryTargets) {
     })(i);
 }
 
+// tslib

```

Comment:
An experimental feature we are adding to support async functions in ES5 and earlier. Instead of emitting the `__extends`, `__awaiter`, and `__generator` functions directly in the output file, the developer can instead include a 'tslib.js' file that contains those functions.

tslib.js uses a UMD (Universal Module Dependency) -like approach that makes those functions available on the global when loaded via `<script>`, or as exported functions when loaded either with CommonJS or AMD. In addition, when importing tslib in Common JS or AMD, you can choose to call a special `install` exported function that installs the helper methods on the global object.

This is an advanced feature and currently experiemental. If a developer choses to use tslib.js, they may pass the `--noHelpers` (experimental) command line option to tsc and it will not emit the helper methods to any file.

For example:
```TypeScript
import { __extends, __awaiter, __generator } from 'tslib';
class B extends A {
}
```
Or:
```TypeScript
import { install } from 'tslib';
install();

class B extends A {
}
```